examples/create_alert.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+
+  tasks:
+  - name: Create an alert in ManageIQ
+    manageiq_alert:
+      miq_url: http://miq.example.com
+      miq_username: admin
+      miq_password: secret
+      miq_verify_ssl: false
+      description: Test Alert 01
+      options:
+        notifications:
+          delay_next_evaluation: 600
```

Comment:
oh that's a bad name for that. I've created a ticket to rename this in manageiq model
https://trello.com/c/dfu1Qumi/138-rename-delay-next-evaluation-in-manageiq